network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/post_tags.html

```diff
@@ -0,0 +1,5 @@
+<div class="">Tagged:
+  {% for tag in page.tags.all %}
+  <strong>{{ tag }}</strong>{% if not forloop.last %}, {% endif %}
```

Comment:
`not forloop.last` I like that

Reply:
I _always_ forget [they exist](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#for) and they are suuuuper handy